### PR TITLE
:book: Bikeshed styling on code blocks in books a bit

### PR DESCRIPTION
**Pull request by DirectXMan12**
This removes the margin before/after collapsible code blocks, so that
collapse-normal-collapse mostly appears as one continuous block.

### CODE REVIEW

**Comment by DirectXMan12** [comment]
(got some comments on the spacing being confusing)

I played around with getting the text to look kinda like a comment, but felt it interrupted the readability too much, happy to have other feedback.

**Comment by apelisse** [comment]
How can one preview this? :-)

**Comment by mengqiy** [comment]
@apelisse In `netlify/kubebuilder/deploy-preview` at the bottom. 
/assign @apelisse

**Comment by apelisse** [comment]
it's an improvement, but I'm still mildly happy with it. When you expand the hidden area, you have litteraly 0 way to know what is the expended thing vs not. Plus you expand something that is in code and shows you out of code comments, it's hard to follow/understand.

**Comment by DirectXMan12** [comment]
I played around with indenting a bit but ultimately left it out.  I'll try re-integrating it and see how that looks
